### ISSUE TITLE
Can't give input

### ISSUE THREAD

**Issue by BlackBearFTW** [botsk/BrainFuck-Interpreter#1]
I tried your website with the current script, but it doesn't ask me for any popup, nor can I do anything with that terminal.

// Input 
,>,

// Sum up values
[<+>-]

// Turn decimal into character
>++++++++[<++++++++++++>-]
<[<->-]

**Comment by TheShark27** [comment]
I'm not the dev, but I'm super interested in learning Brainfuck and I was bummed to see the input field was broken. So I looked into it a little bit.
It turns out that the input field was meant to be summoned by this button:
![image](https://user-images.githubusercontent.com/31541275/155150795-4a12dd92-c036-418d-a7be-e98091732505.png)
However, it doesn't seem to be the case.

I looked into the HTML source code, and that button seems to change the div "input-div" from
`<div id="input-div" style="width: 100%; text-align: center; display: none; position: absolute; bottom: -100px;" bis_skin_checked="1">`
to
`<div id="input-div" style="width: 100%; text-align: center; display: inline-block; position: absolute; bottom: 10px;" bis_skin_checked="1">`

All it does is basically changing `display: none` to `display: inline-block`.
So I tried removing "display: inline-block" altogether and... hooray, the input field appears!
![image](https://user-images.githubusercontent.com/31541275/155151768-683929c8-a715-4e26-92df-1cd0dd3e636b.png)

So, my quick and dirty suggestion to @usaikiran to fix this bug would be to change the code responsible for changing `display: none` to `display:inline-block`, and making it remove `display:none` instead.
